app/javascript/packs/storiesListScript.js

```diff
@@ -0,0 +1,39 @@
+function hasClass(el, cls) {
+  return (
+    el.className && new RegExp(`(\\s|^)${  cls  }(\\s|$)`).test(el.className)
+  );
+}
```

Comment:
If I'm not mistaken, this can be replaced by `element.classList.contains("name of the class")` directly where this function is used

Reply:
did not look at the code, I just change `var` in `const`. Yes I think this old code can be moved. I'm adding this

Reply:
not sure the function makes sense. Do you want to keep it ? @rhymes

Reply:
No, I think replacing with `element.classList.contains()` directly is preferable.